boards/nrf9160dk/include/periph_conf.h

```diff
@@ -28,6 +28,41 @@
 extern "C" {
 #endif
 
+
+/**
+ * @name    I2C configuration
+ * @{
+ */
+static const i2c_conf_t i2c_config[] = {
+    {
+        .dev = NRF_TWIM3_S,
+        .scl = 31,
+        .sda = 30,
```

Comment:
This is 

```suggestion
        .scl = GPIO_PIN(0, 31),
        .sda = GPIO_PIN(0, 30),
```

right?

Reply:
This shouldn't harm.